packages/backend/key-management.sh

```diff
@@ -2,23 +2,43 @@
 
 ACTION="$1"
 SECRET="$3"
+REGION="$4"
 
 if [[ "$2" == "--alchemy" ]]; then
   NAME="nouns/AlchemyKey"
   DESCRIPTION="Alchemy API key"
 elif [[ "$2" == "--executor" ]]; then
   NAME="nouns/ExecutorPrivateKey"
   DESCRIPTION="Ethereum private key with Executor rights to the FOMO Nouns Contract"
+elif [[ "$2" == "--google-api-key" ]]; then
+  NAME="nouns/GoogleApiKey"
+  DESCRIPTION="ReCaptcha: API key associated with the Google Cloud project"
+elif [[ "$2" == "--google-project-id" ]]; then
+  NAME="nouns/GoogleProjectId"
+  DESCRIPTION="ReCaptcha: Google Cloud project ID"
+elif [[ "$2" == "--recaptcha-key" ]]; then
+  NAME="nouns/ReCaptchaKey"
+  DESCRIPTION="ReCAPTCHA key associated with the site/app"
+elif [[ "$2" == "--recaptcha-action" ]]; then
+  NAME="nouns/ReCaptchaAction"
+  DESCRIPTION="ReCAPTCHA: the user-initiated action specified in ReCaptcha call"
```

Comment:
Per above, let's just move this into the `app.js`. No need for this to be stored in KMS.

Reply:
It's now provided as ENV variable to lambda function